CHANGELOG.md

```diff
@@ -8,17 +8,13 @@ this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Changed
 
-- Updated `Conn` event field from `duration` to `end_time` to ensure
-  consistency with other protocol events. The field now represents the
-  session's final timestamp instead of the session duration.
 - Updated `giganto-client` dependency to support type-driven stream
   requests, replacing `NodeType`-based handling with `StreamRequestPayload`
-  enum for improved API consistency. This update also adds `start_time`
-  field to all protocol event structures.
-- Bumped bincode crate to 2.0 and modified the related code.
+  enum for improved API consistency.
```

Comment:
The statement "This update also adds the start_time field to all protocol event structures." is indeed a change introduced after the previous release (0.6.4), so this entry should be retained.

Reply:
I’ve updated it.